### PR TITLE
Draw blocks only for 1s and 0s

### PR DESCRIPTION
**Pull request by jyothishnt**
May be later we could include a color config file to set colours

### CODE REVIEW

File: src/scripts/PhyloCanvas.js [hunk 1]
```diff
@@ -1027,11 +1027,12 @@
 
           if (parseInt(this.data[columnName])) {
```
Comment (richardgoater) [inline src/scripts/PhyloCanvas.js]: What happens if the column data cannot be parsed as a number?

Comment (jyothishnt) [inline src/scripts/PhyloCanvas.js]: It will draw nothing as of now.

Comment (richardgoater) [inline src/scripts/PhyloCanvas.js]: Good old NaN :)

**Comment by richardgoater** [comment]
:+1:

**Pull request merged**